components/history/core/browser/DEPS

```diff
@@ -0,0 +1,3 @@
+include_rules = [
+  "+sql/test",
+]
```

Comment:
missing eol

Reply:
Fixed in 39988cccc84fed28c7acca77ee9f01a18c2530e0